### ISSUE TITLE
add function to synchronize between devices using Google drive or DropBox

### ISSUE THREAD

**Issue by MakuSensei** [inorichi/tachiyomi#2340]
hello and greetings from Argentina
I wanted to request a function to synchronize between several devices using Google drive or DropBox. so as to be more comfortable to exchange device when reading a manga. 
I use tachiyomi on my mobile and on 3 tablets. That is why an online backup and synchronization medium would be useful. 
I am sure that the other users we use this excellent manga app would be very useful. 
Even if they find it interesting, other storage media could be used to perform backup and synchronization. 
for example, that this function is to create and replace one or more files where the information is, and this is modified when opening and closing the application, and in case of a forced closure, carry out synchronization in the background 
so in summary a section on settings within backup, where to choose a service in the cloud, ftp, webdav or a local server. (this at the user's choice)
something like that would be the best idea to implement this function leaving the user their best choice.

**Comment by HI-coder-star** [comment]
Try Autosync for Google Drive & Simple File Manage this work for me

**Comment by Quiksmage** [comment]
> Try Autosync for Google Drive & Simple File Manage this work for me

I don't believe that will work. You'd have to synchronize root level access folders in /data/data for that to work.

**Comment by RuthlessRuler** [comment]
Also, when adding this feature, it would be nice if others like Box/OneDrive can be added too.

**Comment by arkon** [comment]
Duplicate of #912
